annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -499,10 +499,8 @@ private boolean hasAdminRights(Authentication authentication) {
 	    throw new RequestBodyValidationException(annotation, I18nConstants.ANNOTATION_CANT_PARSE_BODY, e);
 	} catch (AnnotationValidationException e) {
 	    throw new RequestBodyValidationException(annotation, I18nConstants.ANNOTATION_CANT_PARSE_BODY, e);
-	} catch (HttpException e) {
-	    // TODO: change this when OAUTH is implemented and the user information is
-	    // available in service
-	    throw e;
+	} catch (ApplicationAuthenticationException e) {
+	    throw new HttpException(e.getMessage(), I18nConstants.USER_NOT_AUTHORIZED, null, HttpStatus.FORBIDDEN, e);
```

Comment:
The handling of the HttpExceptions needs to remain in the code, they must not be wrapped again.  ApplicationAuthenticationException is already a HttpException, it is only required to set the appropriate status when the exception is thrown

Reply:
Done.